packages/fluxible-addons-react/package.json

```diff
@@ -19,11 +19,13 @@
   },
   "peerDependencies": {
     "fluxible": ">=1.0.0",
+    "prop-types": "^15.5.8",
```

Comment:
I think this should be `dependencies` similarly to what was done with `create-react-class`.